### PR TITLE
f-checkout@0.23.0 f-button component

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- Moved justeat packages to the dev dependencies
- Updated f-button component to the latest (0.4.1)
- Renamed button-component to f-button for concistency between projects

### Changed
- Moved justeat packages to the dev dependencies
- Updated f-button component to the latest (0.4.1)
- Renamed button-component to f-button for concistency between projects

### Changed
- Moved justeat packages to the dev dependencies
- Updated f-button component to the latest (0.4.1)

### CODE REVIEW

File: packages/f-registration/package.json [hunk 1]
```diff
@@ -71,6 +66,11 @@
     "@vue/test-utils": "1.1.1",
     "axios-mock-adapter": "1.19.0",
     "flush-promises": "1.0.2",
-    "node-sass-magic-importer": "5.3.2"
+    "node-sass-magic-importer": "5.3.2",
+    "@justeat/f-button": "0.4.1",
+    "@justeat/f-card": "0.7.0",
+    "@justeat/f-error-message": "0.3.1",
+    "@justeat/f-form-field": "1.1.0",
+    "@justeat/f-vue-icons": "1.10.0"
```
Comment (minrwhite) [inline packages/f-registration/package.json]: Can these be put in at the top of `devDependencies`, to keep subsequent PRs clean, as they'll be reordered to there next time someone adds a package via Yarn

File: packages/f-checkout/package.json [hunk 2]
```diff
@@ -73,6 +68,11 @@
     "@vue/cli-plugin-unit-jest": "4.4.6",
     "@vue/test-utils": "1.0.3",
     "node-sass-magic-importer": "5.3.2",
-    "vuex": "3.5.1"
+    "vuex": "3.5.1",
+    "@justeat/f-alert": "0.1.1",
+    "@justeat/f-button": "0.4.1",
+    "@justeat/f-card": "0.8.0",
+    "@justeat/f-error-message": "0.3.0",
+    "@justeat/f-form-field": "1.6.1"
```
Comment (minrwhite) [inline packages/f-checkout/package.json]: > Can these be put in at the top of `devDependencies`, to keep subsequent PRs clean, as they'll be reordered there next time someone adds a package dependency via Yarn

And the same for these.

**Review by AlixRichards — DISMISSED** [review]
There is a tiny typo in two of the changelogs for the word `consistency` - but LGTM 👍

File: packages/f-searchbox/package.json [hunk 3]
```diff
@@ -71,6 +68,9 @@
     "@vue/test-utils": "1.0.3",
     "node-sass-magic-importer": "5.3.2",
     "vue": "2.6.10",
-    "vuex": "3.5.1"
+    "vuex": "3.5.1",
+    "@justeat/f-button": "0.4.1",
+    "@justeat/f-error-message": "0.3.1",
+    "@justeat/f-vue-icons": "1.13.0"
```
Comment (minrwhite) [inline packages/f-searchbox/package.json]: > Can these be put in at the top of `devDependencies`, to keep subsequent PRs clean, as they'll be reordered to there next time someone adds a package via Yarn

And same for these

File: packages/f-checkout/src/components/Checkout.vue [hunk 4]
```diff
@@ -43,21 +43,21 @@
 
                 <user-note data-test-id="user-note" />
 
-                <button-component
+                <f-button
```
Comment (ashleynolan) [inline packages/f-checkout/src/components/Checkout.vue:46]: Can we name this `button-component` and import `import ButtonComponent from '@justeat/f-button';`?

Think it reads a little better then in the template (and we don't use `f-` generally when importing components elsewhere).

File: packages/f-registration/src/components/Registration.vue [hunk 5]
```diff
@@ -135,15 +135,15 @@
                     </template>
                 </form-field>
 
-                <button-component
+                <f-button
```
Comment (ashleynolan) [inline packages/f-registration/src/components/Registration.vue:138]: Same here

File: packages/f-checkout/src/components/Checkout.vue (hunk 4 above)
Comment (dandel10n) [reply packages/f-checkout/src/components/Checkout.vue:46]: FButton/f-button looked to match one of [the recommended naming schemes](https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended) and I changed it as the result of discussion in https://github.com/justeat/fozzie-components/pull/507

**Review by minrwhite — DISMISSED** [review]

**Review by minrwhite — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

File: packages/f-checkout/src/components/Checkout.vue (hunk 4 above)
Comment (dandel10n) [reply packages/f-checkout/src/components/Checkout.vue:46]: I will merge this for now and can change later if the team thinks `...-component` is better to use

**Pull request merged**